AsyncDisplayKit.xcodeproj/project.pbxproj

```diff
@@ -2013,6 +2016,21 @@
 			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-AsyncDisplayKitTests/Pods-AsyncDisplayKitTests-resources.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
+		47FEBAC527C86C3C7E855E41 /* 📦 Embed Pods Frameworks */ = {
```

Comment:
You are probably on CocoaPods version 1.0 and you should update to 1.0.1 so we do not add the emojis back in to the project settings. Not sure if we have to add these kind of settings at all ...